app/models/miq_worker/runner.rb

```diff
@@ -142,6 +142,7 @@ def starting_worker_record
 
   def started_worker_record
     reload_worker_record
+    @worker.sd_notify_started if @worker.systemd_worker?
```

Comment:
Don't love doing this here, feels like we're going up to the worker instance for all of these so maybe moving all of this info `@worker.started!` or something would be better?  Just testing having systemd checking starting/stopping/watchdog timeout and this seemed to be the place where we note that something is started

Reply:
I'm hoping we can find all the places we're doing systemd/podified related checks and move them up instead of deep in methods like this.  I don't know if we'll be able to do this until we have more implemented differently in each process management system.